docs/guide/machine-learning.asciidoc

```diff
@@ -38,9 +38,8 @@ model in {es}.
 [[ml-nlp-pytorch]]
 === Natural language processing (NLP) with PyTorch
 
-
-IMPORTANT: You need to use PyTorch `1.13` or earlier to import an NLP model. 
-Run `pip install torch==1.13` to install the aproppriate version of PyTorch.
+IMPORTANT: You need to appropriate version of PyTorch to import an NLP model. Run
+`python -m pip install 'eland[pytorch]'` to install the latest version.
```

Comment:
It's never the latest version, so I'm suggesting this tweak:

```suggestion
IMPORTANT: You need to appropriate version of PyTorch to import an NLP model. Run
`python -m pip install 'eland[pytorch]'` to install that version.
```

Also, README.md has the same wording, would you mind updating it too?

Reply:
Also there's a missing word that I missed earlier, sorry. You need to **install the** appropriate version.

Reply:
Changed it in both files! thank you